website/source/docs/vault-integration/index.html.md

```diff
@@ -236,12 +236,23 @@ token_renewable true
 token_policies  [default nomad-server]
 ```
 
-`-orphan` is included above to prevent revocation of the token when its parent expires. 
-See the [Vault token hierarchy](https://www.vaultproject.io/docs/concepts/tokens.html#token-hierarchies-and-orphan-tokens) 
-documentation for more information.
+The `-orphan` flag is included when generating the Nomad server token above to 
+prevent revocation of the token when its parent expires. Vault typically 
+creates tokens with a parent-child relationship. When an ancestor token is revoked, all of its descendant tokens and their associated leases are revoked as well.  
+
+When generating Nomad's Vault token, we need to ensure that revocation of the 
+parent token does not revoke Nomad's token. To prevent this behavior we 
+specify the `-orphan` flag when we create the Nomad's Vault token. All 
+other tokens generated by Nomad for jobs will be generated using the policy 
+default of `orphan = false`
+
+More general information about creating orphan tokens can be found in the 
+Vault documentation, specifically at 
```

Comment:
`found in [Vault's Token Hierarchies and Orphan Tokens documentation][tokenhierarchy].`